src/components/CreateAddress/PublicKeyImporter.jsx

```diff
@@ -170,6 +183,7 @@ class PublicKeyImporter extends React.Component {
         <TextPublicKeyImporter
           publicKeyImporter={publicKeyImporter}
           validateAndSetPublicKey={this.validateAndSetPublicKey}
+          addressType={addressType}
```

Comment:
This modification is necessary because we must pass `addressType` to the `TextPublicKeyImporter`.

Our suspicion is that none of the *other* changes in this file are necessary.